jonathanyeong/smart-shopping-list
# Connect to firestore DB - read and write to it

## Description
Reads and writes to Firestore DB. It will only read from the DB on first render of the app.

## Related Issue



## Acceptance Criteria


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="2032" alt="Screen Shot 2021-04-05 at 11 58 20 pm" src="https://user-images.githubusercontent.com/3861088/113671140-d2119700-966a-11eb-9979-057ff3f5432d.png">

### After

<img width="1283" alt="Screen Shot 2021-04-05 at 11 56 18 pm" src="https://user-images.githubusercontent.com/3861088/113671150-d5a51e00-966a-11eb-88e0-de5ee7ca3d05.png">
<img width="407" alt="Screen Shot 2021-04-05 at 11 56 13 pm" src="https://user-images.githubusercontent.com/3861088/113671153-d5a51e00-966a-11eb-882a-56e9f0db6ce6.png">

## Testing Steps / QA Criteria

1. Add an element to firestore DB - to the shoppingLists collection.
2. Run the react app you should see the element you added appear on the page.
3. Click the button to add a new shopping list document.
4. Go to the firestore DB and see the element appear in the shoppingLists collection. It will always have 3 elements - beer, coffee, milk.
